tests/RequestHandler.spec.ts

```diff
@@ -24,14 +24,19 @@ describe('RequestHandler', () => {
     }
     blockchain.setLaskBlock(mockLastBlock);
 
-    const createRequest = readFileSync('./tests/requests/create.json');
+    // Read create operation request from file.
+    const requestString = readFileSync('./tests/requests/create.json', 'utf8');
+    const requestStringWithoutCarriageReturn = requestString.replace(/[\r]+/g, ''); // Remove all Carriage Returns in case of file on Windows.
```

Comment:
Instead of handling line endings in the code, can we force git to store file in either Windows or Linux format to avoid this ambiguity?

Reply:
Hi @sylar217, great suggestion, let me try play around with it.